includes/class-wp-job-manager-post-types.php

```diff
@@ -301,7 +302,7 @@ public function register_post_types() {
 			'expired',
 			array(
 				'label'                     => _x( 'Expired', 'post status', 'wp-job-manager' ),
-				'public'                    => true,
+				'public'                    => ! isset( $_GET['s'] ),
```

Comment:
Would you mind adding a comment to your core trac issue above this line?
Something like:
```
// For background and to track issue, see https://core.trac.wordpress.org/ticket/44737.
```